myp5.js

```diff
@@ -51,44 +47,53 @@ function draw() {
   }  
 }
 
-function lyrics(x, y) {
+function loadLyricsOnce() {
+
+  for (var i = 0; i < lyrics.length; i++) {
+    lyricObjects[i] = new Lyric(random(width), random(height), lyrics[i]);
+  }
+
+}
+
+function Lyric(x, y, _text) {
   this.acceleration = createVector(0, 0);
   this.velocity = p5.Vector.random2D();
```

Comment:
Be very careful with your variable/class names! Remember that a class is a "cookie cutter" and it makes a singular thing, hence `Lyric` over `lyrics`